app/controllers/api/v1/site_settings_controller.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+module Api
+  module V1
+    class SiteSettingsController < ApiController
+      # GET /api/v1/site_settings/:name
+      def show
+        render_json data: SettingGetter.new(setting_name: params[:name], provider: 'greenlight').call # TODO: - ahmad: fix provider
```

Comment:
I feel that this is similar somehow to the `MeetingOption` and `RoomMeetingOption` case from a data point of view.
With that said, I think that we can (even for consistency) do the join in a method inside of `Setting|SiteSetting` model `get_setting(name:, provider:)`.

Reply:
Do we need a `SiteSettingController#show` API?
Because according to the mockups we'd be having a single page where we can edit all site settings individually like with the room settings.
In such case and because the `siteSetting` model isn't that complex to require a specific endpoint to fetch it, we can create only `SiteSettingController#index` that fetches all `siteSettings` for the provider with settings `:name` and `:value`.
And we'd have also the `Admin::SiteSettingController#update` that will update a specific site setting.

Reply:
Whoever is creating the page in the admin panel can create a `SiteSettings#index` that will return all those values